### PR TITLE
Restart ratelimiter on timeout

### PR DESCRIPTION
**Pull request by tspenov**
## Changes

That will fix the case where `{:error, :timeout}` comes for some reason from the RateLimiter gen server due to some congestion. Or will rule it out.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -122,7 +122,18 @@ config :sasl, sasl_error_logger: false
...
+    :request_id,
+    :api_token,
+    :user_id,
+    :remote_ip,
+    :complexity,
+    :query,
+    :san_balance,
+    :shard,
+    :guild,
+    :channel
```
Comment (IvanIvanoff) [inline config/config.exs:135]: Does adding this affect the rest of the logs?

File: lib/sanbase/discord/discord_consumer.ex [hunk 2]
```diff
@@ -312,8 +312,25 @@ defmodule Sanbase.DiscordConsumer do
...
+        log(msg, "WARM UP TIMEOUT. Ratelimiter pid: #{pid}", type: :error)
+
       error ->
         log(msg, "WARM UP ERROR #{inspect(error)}", type: :error)
     end
   end
+
+  def restart_ratelimiter() do
+    supervisor_pid =
+      Process.whereis(Ratelimiter)
```
Comment (IvanIvanoff) [inline lib/sanbase/discord/discord_consumer.ex:327]: You do not have an alias for this, shouldn't this be `Nostrum.Api.Ratelimiter` instead?

Comment (tspenov) [reply lib/sanbase/discord/discord_consumer.ex:327]: It has: `@registered_name Ratelimiter`: https://github.com/Kraigie/nostrum/blob/master/lib/nostrum/api/ratelimiter.ex#L26

File: config/config.exs (hunk 1 above)
Comment (tspenov) [reply config/config.exs:135]: It does not, it just not shows them

Comment (tspenov) [reply config/config.exs]: though removed them, because they are not very helpful

**Pull request merged**